common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -40,6 +41,18 @@ public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
   }
 
   /**
+   * Constructs the worker net address from thrift construct.
+   *
+   * @param netAddress the thrift net address
+   */
+  public WorkerNetAddress(NetAddress netAddress) {
```

Comment:
I see, it does seem specific to the worker ... by the same logic, it might make sense to rename the Thrift structure to WorkerNetAddress ... on a related note, it seems odd to be calling a hostname and three ports an "address" ... something like WorkerNetConf might make more sense